source/common/network/io_socket_handle_impl.h

```diff
@@ -85,6 +86,7 @@ class IoSocketHandleImpl : public IoHandle, protected Logger::Loggable<Logger::I
 
   os_fd_t fd_;
   int socket_v6only_{false};
+  absl::optional<int> domain_;
```

Comment:
nit: const

Reply:
thanks! will change

Reply:
fixed in c1d4d36567caae65137fe36f66fe188c0622533c